frontend/templates/views/partials/teaser.j2

```diff
@@ -120,7 +120,7 @@
     <ul class="ap-m-teaser-info-list">
       {% for used_component in teaser_doc.used_components %}
       <li class="ap-m-teaser-info-list-item">
-        {% set component_path = '/content/amp-dev/documentation/components/reference/' + used_component + '.md' %}
+        {% set component_path = '/content/amp-dev/documentation/components/reference/' + used_component + '-v0.1.md' %}
```

Comment:
Is it a good idea to hard-code v0.1?

Reply:
No not really. Just brought it back to the state it was before but I am planning to improve this.